sealer/cmd/delete.go

```diff
@@ -22,13 +22,30 @@ import (
 	"github.com/spf13/cobra"
 )
 
+const (
+	yes = "y"
+	no  = "n"
+)
+
 // deleteCmd represents the delete command
 var deleteCmd = &cobra.Command{
 	Use:     "delete",
 	Short:   "delete a cluster",
 	Long:    `if provider is BARESERVER will delete kubernetes nodes, or if provider is ALI_CLOUD, will delete all the infra resources`,
 	Example: `sealer delete -f /root/.sealer/mycluster/Clusterfile`,
 	Run: func(cmd *cobra.Command, args []string) {
+		var input string
+		for {
```

Comment:
1. "y" or "yes"
2. We also can provide an --force flag to delete cluster by force.

Reply:
LGTM .   @tea403 

a regex using may be ok.  

```
package main

import (
	"fmt"
	"regexp"
)

func main() {

	var yesRx = regexp.MustCompile("^(?i:y(?:es)?)$")

	var input string
	for {
		fmt.Scanln(&input)
		if yesRx.MatchString(input) {
			fmt.Println("regex match yes")
			break
		} else {
			fmt.Println("regex match not")
		}
	}
}
```

Reply:
i already recommit the pr